static/app/components/events/eventTagsAndScreenshot/dataSection.tsx

```diff
@@ -1,60 +0,0 @@
-import styled from '@emotion/styled';
-import kebabCase from 'lodash/kebabCase';
-
-import GuideAnchor from 'sentry/components/assistant/guideAnchor';
-import {
-  EventDataSection,
-  SectionContents,
-} from 'sentry/components/events/eventDataSection';
-import QuestionTooltip from 'sentry/components/questionTooltip';
-import space from 'sentry/styles/space';
-
-type Props = {
-  children: React.ReactNode;
-  description: string;
-  title: string;
-  className?: string;
-};
-
-export function DataSection({title, description, children, className, ...props}: Props) {
-  const type = kebabCase(title);
```

Comment:
I validated that all the GuideAnchor target's being passed in this way were not used